arma/arma3/egg-arma3.json

```diff
@@ -2,9 +2,9 @@
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PANEL",
     "meta": {
         "version": "PTDL_v2",
```

Comment:
This is not pelican exported or a realy old version

Reply:
I can't get file uploads (of any kind) to work on my current Pelican deployment in Docker. Would you mind possibly importing the Pterodactyl version on yours, exporting it as Pelican, and committing it to the branch? On a side note, unless this is already in the works, I might try to develop a Python script to convert Eggs both ways. This way, contributors that don't have both panels can easily make PRs that update both Eggs reliably.